index-handlers/src/main/java/no/sikt/nva/nvi/index/model/search/OrderByFields.java

```diff
@@ -0,0 +1,24 @@
+package no.sikt.nva.nvi.index.model.search;
+
+import java.util.Arrays;
+
+public enum OrderByFields {
+    CREATED_DATE("createdDate");
+
+    private final String value;
+
+    OrderByFields(String value) {
+        this.value = value;
+    }
+
+    public static OrderByFields parse(String value) {
+        return Arrays.stream(OrderByFields.values())
+                   .filter(field -> field.getValue().equalsIgnoreCase(value))
+                   .findFirst()
+                   .orElseThrow(() -> new IllegalArgumentException("Invalid OrderByField: " + value));
```

Comment:
Don't repeat the input value, just present the info that it is not allowed and tell which ones are

Reply:
This prevents a security issue for us and consumers

Reply:
Good catch!